### PR TITLE
Minor tweaks 2

### PR DESCRIPTION
**Pull request by osiro**
Changes:

- [x] using `rails_helper` instead of `spec_helper`
- [x] removing controller users, actions edit/update are already provided by devise (no need to duplicate logic)
- [x] using letter opener for dev env
- [x] adding configs to speed up feature tests

### CODE REVIEW

File: app/controllers/devise_customisations/registrations_controller.rb [hunk 1]
```diff
@@ -6,4 +6,11 @@ def build_resource(args)
     end
   end
 
+protected
+
+  def after_update_path_for(resource)
+    # after user edits their own profile, we redirect to the same place
+    # user's get redirect when sign in (usually dashboard)
```
Comment (jordanmaguire) [inline app/controllers/devise_customisations/registrations_controller.rb]: :+1:

File: README.md [hunk 2]
```diff
@@ -44,5 +44,5 @@ The output of this script should give you the instructions you will need.
 - (Possible) Add in rake task for adding build to CI
 - Add generators for CRUD
 - Use dropdown menu for user actions
-- Add in mailer defaults for dev (mailcatcher), staging, and production (jazz)
+- Add in mailer defaults for staging, and production (jazz)
```
Comment (jordanmaguire) [inline README.md]: Put the corresponding change in the list of things the template does. EG: Provides letter opener for mail stuffs

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  config.include ControllerParameterSupport, type: :controller
+  config.extend ControllerDescribeAssignsHelper, type: :controller
+
+  # Features
+  config.include FeatureHelper, type: :feature
+  config.include FeatureAuthenticationHelper, type: :feature
+  config.include FeatureNavigationHelper, type: :feature
+  config.extend FeatureAuthenticationMacros, type: :feature
+  config.include Warden::Test::Helpers
+end
```
Comment (jordanmaguire) [inline spec/rails_helper.rb]: :+1:

**Comment by jordanmaguire** [comment]
Lots of good changes here :+1:

File: README.md (hunk 2 above)
Comment (osiro) [inline README.md]: done!

**Pull request merged**